index.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
```

Comment:
In this library, we're putting semicolons everywhere. It would be great if you can add those semicolons after every statement. Also, after the types (Line 5 to 9).  Once it is done, I'll directly merge this. Thanks!

Reply:
Did you paste this code in pastebin? If so, why am I not able to see the semicolons?🤔

Reply:
I was unable to commit to it (it is not my pr nor my branch), so I thought that the easiest solution is to give you the code.

Reply:
Ohh thanks... The only option I have now is to create a new branch, merge this branch to the new branch, and add these changes.

Reply:
FYI: maintainers can edit this branch directly, but non-maintainers can't

Reply:
Awesome! Thanks... Merging this.